autocertkit/utils.py

```diff
@@ -65,6 +65,7 @@
 
 # Capability Tags
 REQ_CAP = "REQ"
+REQ_CAP_MULTICAST = "REQ_MULTICAST"
```

Comment:
@bpwook 
From @rdobson 

> if it is run, and passes, the 'multicast' flag should be indicated in the results

With this change, the result.txt will shows  "REQ_MULTICAST: Supported"